lib/index.js

```diff
@@ -10,7 +10,7 @@ var internals = {
 internals.store.lookUp = function (obj, method) {
   var i = this.length;
   var lookup;
-  while (i-- && !lookup) {
+  while (!lookup && i--) {
```

Comment:
I guess we have to agree to disagree. I can't take that statement seriously given it's use in everything from Node core to hapi :-)